zim/__init__.py

```diff
@@ -191,7 +191,7 @@
 	# Windows specific environment variables
 	# os.environ does not support setdefault() ...
 	if not 'USER' in os.environ or not os.environ['USER']:
-		os.environ['USER'] = os.environ['USERNAME']
```

Comment:
Good catch to use `get()` here to avoid exceptions. However, would prefer to keep it defaulting to `None`. Having `None` value if also `USERNAME` is not available allows the fallback in line 209 to handle it.